server/src/repositories/asset.repository.ts

```diff
@@ -303,6 +294,7 @@ export class AssetRepository implements IAssetRepository {
   @Chunked()
   async restoreAll(ids: string[]): Promise<void> {
     await this.repository.restore({ id: In(ids) });
+    await this.updateAll(ids, { trashReason: null });
```

Comment:
Why not just work with `undefined` in this case as well? IMO `this.updateAll(ids, {})` is a bit simpler

Reply:
Nvm, has been discussed on Discord.